src/utils.jl

```diff
@@ -332,7 +329,7 @@ end
 compact_mem(x) = x
 compact_mem(x::StringArray{String}) = convert(StringArray{WeakRefString{UInt8}}, x)
 
-#nonmissing(::Type{Union{Missing, T}}) where T = T
+nonmissing(::Type{Union{Missing, T}}) where T = T
```

Comment:
`Missings.T`  and `Base.nonmissingtype` already do this (but also work with any type, being a no-op).

Reply:
Thanks for the pointer!  I did not know about `Base.nonmissingtype`.